epoch/api/channels_api_usage.md

```diff
@@ -518,3 +521,268 @@ details:
+```
+{'action': 'on_chain_tx',
+ 'payload': {'tx': 'tx$3cDMp6242sBycND2FPT2jcDWFceRgA7zL3ckU8VzLQdvf2Uqjx5CKkjMXbYrmYjMnLnDihVTrF2fCLqNG93BTAsCNWT6QiJwdmXrTXLQ2d2d7rAc5bYepTC2w3LyrZ37jhx3dN7ATusjXtSu6jw9N8exaPxnjKD3twye5B6bdqbZEHKXXtqStUmaTmUDofEWtGaUCxTWKCboMH7T2mxEjzxgpaH2fbHRxA3GmxaTaKWoTfbnvqragH9QVo6QxiCRAGNUEkbRbPw8m1qPUKjVFWWQSZ9VcCdXte3DitS3anXv7jtTKAA7uuj5pbdG4qi64dDLTd52sSQP6JZpzMxa6oyJTDo5s'
+            }
+}
+```
+They are both to compute the transaction hash. Using it,
+participants can track its progress on the chain: entering the mempool, block
+inclusion and a number of confirmations.
+
+After the `minimum_depth` block confirmations each participant is informed
+for the deposit progress on-chain by one's own node:
+```
+{'action': 'info',
+ 'payload': {'event': 'own_deposit_locked'}
+ }
+```
+An update from one's own node that the other party had confirmed that the block
+height needed is reached:
+```
+{'action': 'info',
+ 'payload': {'event': 'deposit_locked'}
+ }
+```
+With this the deposit sequence is complete and the channel can continue with
+other updates. Note that the deposit transaction's `round` and `state_hash`
+are the ones considered latest from the channel's perspective. For example the
+next correct off-chain update/deposit/withdrawal shall have a deposit
+transaction's `round` plus one.
+
+### Withdraw events
+
+After the channel had been opened any of the participants can initiate a
+withdrawal. The process closely resembles the [update](#update). The most notable
+difference is that the transaction has been co-signed: it is `channel_withdraw_tx` and
+after the procedure is finished - it is being posted on-chain.
+
+Since both the initiator and responder can withdraw tokens, in the scope of this description we
+will call the participant that commits tokens to the channel a withdrawer and
+the other party - an acknowledger. Note that any public key outside of the channel participants
+is considered invalid for the withdrawer role.
+
+#### Withdraw transaction
+The `channel_withdraw_tx` is a change to be applied on top of the latest channel state. It also is
+posted on-chain and is included in a block. It has the following structure:
+
+  | Name | Type | Description |
+  | ---- | ---- | ----------- |
+  | channel id | string | ID of the channel|
+  | to | string | withdrawer's public key |
+  | amount | integer | the amount taken out from the channel |
+  | ttl | integer | minimum block height to include the transaction |
+  | fee | integer | fee to be paid to the miner |
+  | state_hash | string | the root of the internal channel state hash after the withdraw |
+  | round | integer | the next channel round |
+  | nonce | integer | withdrawer's nonce |
+
+#### Start withdraw
+
+Any of the participants can initiate a withdrawal. The only requirements are:
+* Channel is already opened
+* No off-chain update/deposit/withdrawal is currently being performed
+* Channel is not being closed or in a solo closing state
```

Comment:
* The withdrawal amount must be equal to or greater than zero, and cannot exceed the available balance on the channel (minus the `channel_reserve`).